src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -10,6 +10,8 @@
 public static class TimeZoneInfoTests
 {
     private static readonly bool s_isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+    private static readonly bool s_isOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```

Comment:
does it make sense to move the Linux test code into its own methods and mark such methods with the Platform specific attribute? I think this will make the code cleaner and removing many of the runtime checks. of course we can do the same with Windows specific tests. what you think?